main.go

```diff
@@ -57,6 +57,7 @@ func main() {
 		}
 
 		kafkaConfig["security.protocol"] = kafkaSecurityProtocol
+		kafkaConfig["ssl.ca.location"] = kafkaSslCACertFile // CA certificate file for verifying the broker's certificate.
```

Comment:
Thanks for your contribution @heldertsantos. Are you sure this works? I think you should also include this new environment var in config.go, here https://github.com/Telefonica/prometheus-kafka-adapter/blob/d1cbd70c213248b3597adbdd4d2ddbf6173b47bd/config.go#L111-L113= so the var `kafkaSslCACertFile` gets populated.

Reply:
ping @heldertsantos